### PR TITLE
Change to Plausible

### PR DESCRIPTION
**Pull request by Kongkille**
Changes analytics to Plausible, which is a lightweight GA alternative.

Note that Plausible strips all url query parameters, meaning wind/solar layers won't show up.

Furthermore, it seemed GA also tracked some kind of execution/load time, which I have removed, as I don't think it was used, and I am not sure Plausible supports the same thing.

### CODE REVIEW

File: web/views/pages/index.ejs [hunk 1]
```diff
@@ -99,8 +99,8 @@
         <script defer src="<%= resolvePath('dist/vendor.' + vendorHash + '.js') %>"></script>
         <script defer src="<%= resolvePath('dist/bundle.' + bundleHash + '.js') %>"></script>
 
-        <!-- Google Analytics -->
-        <script async src="https://www.googletagmanager.com/gtag/js?id=UA-79729918-1"></script>
+        <!-- Plausible -->
+        <script defer data-domain="app.electricitymap.org" src="https://plausible.io/js/plausible.local.js"></script>
```
Comment (madsnedergaard) [inline web/views/pages/index.ejs:103]: Probably shouldn't still be local? :)

**Comment by madsnedergaard** [comment]
I made some changes to the thirdparty code in https://github.com/electricitymap/electricitymap-contrib/pull/3854 which are now causing a merge conflict :)

File: web/views/pages/index.ejs (hunk 1 above)
Comment (madsnedergaard) [reply web/views/pages/index.ejs]: Added this comment about testing on mobile apps @Kongkille

Comment (Kongkille) [reply web/views/pages/index.ejs]: Ah great point. We should test if the app can send on localhost

**Comment by Kongkille** [comment]
Just tested on the mobileapp, and it also seems to work. We just have to keep the localhost setting as well. However, it's disabled unless it's in production, so it should not fire when developing locally on the web app.

**Review by madsnedergaard — APPROVED** [review]
Awesome! 🥳

**Comment by madsnedergaard** [comment]
FYI There's some cleanup on mobileapps of removing analytics, but we should address that in a separate PR :)

**Pull request merged**